doc/Container_Deployment.md

```diff
@@ -0,0 +1,89 @@
+# Container based deployment
+
+The provided [Dockerfile](../container/Dockerfile) can be used to
+build a SUSE Linux Enterprise 15 based container image, leveraging
+[entrypoint.bash](../container/entrypoint.bash) script as the entry
+point to ensure that the container runtime environment is set up
+correctly before running the `scc-hypervisor-collector` tool.
+
+## Bind Mount Volumes
+
+The container expects that a properly setup directory will be bind
+mounted on `/var/lib/scchvc`.
+
+This directory should contain the following:
+
+* a `.config/scc-hypervisor-collector` sub-directory, with desired
+  configuration settings.
+* optionally a `.ssh` sub-directory with relevant SSH private key
+  supporting passwordless access to any `qemu+ssh` protocol based
+  Libvirt connection URIs specified in the configuration settings.
+* optionally a `certs` directory containing any certs that are
```

Comment:
Currently we do not use -r for copying the certs. Should we mention that the certificate files need to be placed in certs folder and not be in sub-directories?

Reply:
Good catch - updated entrypoint.sh to do a cp -a ;-)